src/tags/core.js

```diff
@@ -23,10 +23,18 @@ export const nullObj = {
     ctx.wrapScalars ? new Scalar(null) : null,
   default: true,
   tag: 'tag:yaml.org,2002:null',
+  sourceStr: null,
```

Comment:
I don't think this has any role anymore?

Reply:
yes, removed.